src/com/twitter/intellij/pants/service/project/model/ProjectInfo.java

```diff
@@ -182,10 +182,10 @@ public String toString() {
   }
 
   public void merge(ProjectInfo projectInfo) {
-    getLibraries().putAll(projectInfo.getLibraries());
-    getTargets().putAll(projectInfo.getTargets());
-    if (getPythonSetup() == null && projectInfo.getPythonSetup() != null) {
-      python_setup = projectInfo.getPythonSetup();
+    libraries.putAll(projectInfo.libraries);
+    targets.putAll(projectInfo.targets);
```

Comment:
This should be something like 
`libraries.putAll(projectInfo.getLibraries())` as `libraries` is an internal variable to we want to modify, `projectInfo` is from "outside" which we just want to read only from.